src/sentry/mediators/sentry_app_installation_tokens/creator.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import
+
+
+from datetime import datetime
+
+from sentry.utils.cache import memoize
+from sentry.mediators import Mediator, Param
+from sentry.models import (
+    AuditLogEntryEvent, ApiToken, SentryAppInstallationToken
+)
+
+
+class Creator(Mediator):
+    sentry_app_installation = Param('sentry.models.SentryAppInstallation')
+    expires_at = Param(datetime.date, default=None, required=False)
+    # analytics and audit params
+    generate_audit = Param(bool, default=False)
+    user = Param('sentry.models.User')
+    request = Param('rest_framework.request.Request', required=False)
+
+    def call(self):
+        self._create_api_token()
+        self._create_sentry_app_installation_token()
+        return self.sentry_app_installation_token
```

Comment:
I'm not sure if this matters too much but is there any reason we don't want to just return the token?
```python
return self.sentry_app_installation_token.api_token
```

Reply:
@MeredithAnya This way we can get the `sentry_app_installation_token` more easily if we need it. Otherwise, we'd need to do a query to grab it which is annoying.

Reply:
Just trying to think _why_ would need it. I don't have strong feelings on this, just thought I'd mention it